src/CMakeLists.txt

```
@@ -17,6 +17,7 @@ add_library ("${PROJECT_NAME}_lib" STATIC modules/clustering/hierarchical_cluste
 target_link_libraries ("${PROJECT_NAME}_lib" PUBLIC seqan3::seqan3)
 target_link_libraries ("${PROJECT_NAME}_lib" PUBLIC fastcluster)
 target_include_directories ("${PROJECT_NAME}_lib" PUBLIC ../include)
+target_compile_options ("${PROJECT_NAME}_lib" PUBLIC "-pedantic" "-Wall" "-Wextra" "-Werror")
```

I read that you should not use Werror, because why not get warnings as warnings, than errors: https://embeddedartistry.com/blog/2017/05/22/werror-is-not-your-friend/
```suggestion
target_compile_options ("${PROJECT_NAME}_lib" PUBLIC "-pedantic" "-Wall" "-Wextra")
```

---

There are many reasons to use `-Werror` (the linux kernel tried to set this flag as default recently, too).

But it is quite annoying for developing; I removed it.